src/v3/test/integration/__snapshots__/authenticator-apple-sso-extension.test.tsx.snap

```diff
@@ -41,7 +42,7 @@ exports[`authenticator-apple-sso-extension calls verify then cancels the transac
                     class="MuiBox-root emotion-7"
                   >
                     <h2
-                      class="MuiTypography-root MuiTypography-h3 emotion-8"
+                      class="MuiTypography-root MuiTypography-h4 emotion-8"
```

Comment:
Is this style class change in the snapshots due to the font size update Alex put in but showing as a diff now because we're targeting 0.2?